devito/ir/stree/tree.py

```diff
@@ -54,7 +54,11 @@ def dim(self):
 
     @property
     def limits(self):
-        return self.interval.limits
+        return (self.dim.extreme_min, self.dim.extreme_max, 1)
```

Comment:
Where is the `1` coming from here?

Reply:
it's the step size. Dimensions (and SubDimensions) imply a step size of 1 (ie, ++ or --)

some subclasses (e.g., ModuloDimension) behave differently